environments/dev/terraform/clinvar.tf

```diff
@@ -13,6 +13,7 @@ module clinvar {
 
   project_name = "broad-dsp-monster-clingen-dev"
   is_production = false
+  command_center_argo_account_email = "TO DO"
```

Comment:
@danxmoran if we apply the terraform, won't it actually try to create IAM bindings for this?